src/redux/tests/thunks.test.js

```diff
@@ -0,0 +1,30 @@
+import { fetchCourses } from "../courses/asyncActions";
+import { fetchLessons } from "../lessons/asyncActions";
+
+jest.setTimeout(15000);
+
+describe('courses thunk', () => {
+    it('should fetchCourses with correct structure', async () => {
+        jest.mock('axios');
+
+        const dispatch = jest.fn();
+        const thunk = fetchCourses();
+        await thunk(dispatch);
+
+        const { calls } = dispatch.mock;
+        expect(calls).toHaveLength(2);
+    })
+})
+
+describe('lessons thunk', () => {
+    it('should lessons with correct structure', async () => {
+        jest.mock('axios');
+
+        const dispatch = jest.fn();
+        const thunk = fetchLessons();
+        await thunk(dispatch);
+
+        const { calls } = dispatch.mock;
+        expect(calls).toHaveLength(2);
```

Comment:
I suggest you can easily add tests for action types and payloads: instead of only checking the number of calls made to the dispatch function, it's better to test the action types and payloads dispatched, this way the test will be of higher quality :)